api_tests/src/shared.ts

```diff
@@ -66,23 +66,23 @@ export interface API {
 }
 
 export let alpha: API = {
-  client: new LemmyHttp('http://localhost:8541'),
+  client: new LemmyHttp('http://127.0.0.1:8541'),
```

Comment:
localhost on my computer now resolves to `::1` (IPv6) for some reason, and nodejs cant handle that. So i had to change it to the plain ipv4 address.